SpringProject/src/main/java/com/gradproject2019/conferences/service/ConferenceServiceImpl.java

```diff
@@ -24,11 +26,12 @@
 
     private ConferenceRepository conferenceRepository;
 
-    private AuthServiceImpl authServiceImpl;
+    private AuthService authService;
```

Comment:
Curious as to why this has changed from AuthServiceImpl to AuthService?

Reply:
Harry pointed out that we should always be referring to the Service layer through its interface and not the implementation

Reply:
I agree, otherwise what's the point having an interface and an implementation. The implementation might be replaced with another one at any point.